.github/workflows/update_disabled_tests.yml

```diff
@@ -63,13 +68,28 @@ jobs:
           user_name: "Pytorch Test Infra"
           commit_message: "Updating disabled jobs"
 
+      - name: Push unstable jobs to test-infra repository
```

Comment:
Do we really need to push these stats to the test-infra repository? Does it have any consumers?

What if we only uploaded them to s3 instead?

Reply:
Good point!  They are pushed to test-infra for historical reason as the CI use to get them there.  @clee2000 Could we remove this step or is there anything else you have in mind that still need it?  We can update all the docs to point to S3 links instead

Reply:
I'm drafting another PR to clean this up